scripts/compile.ts

```diff
@@ -1,9 +1,9 @@
 #!/usr/bin/env -S deno run --allow-run --allow-read=. --allow-write=./dist
 
-import { sh } from '../src/deps.ts'
+import { shell } from '~/utils.ts';
```

Comment:
What is the reason you decided to not use the dependency?

Reply:
Yeah, I removed `miniexec`; its outdated Deno.run() and improper Deno.Command usage caused stream handling errors, leading to vendor import issues when bundled. Also couldn't find `jsr:` for it.

So just copied this simple `sh` function from `miniexec` and moved into `utils` as `shell`: https://github.com/okTurtles/chel/blob/038b71bcdfb64def7688e896917f52bbc0a0ec16/src/utils.ts#L150